horizons/ai/aiplayer/productionbuilder.py

```diff
@@ -263,24 +263,24 @@ def display(self):
 			return
 
 		tile_colors = {
-			BUILDING_PURPOSE.ROAD:            ( 30,  30,  30),
+			BUILDING_PURPOSE.ROAD:            (+30,  30,  30),
 			BUILDING_PURPOSE.FISHER:          (128, 128, 128),
-			BUILDING_PURPOSE.LUMBERJACK:      ( 30, 255,  30),
-			BUILDING_PURPOSE.TREE:            (  0, 255,   0),
+			BUILDING_PURPOSE.LUMBERJACK:      (+30, 255,  30),
+			BUILDING_PURPOSE.TREE:            (000, 255,   0),
 			BUILDING_PURPOSE.FARM:            (128,   0, 255),
 			BUILDING_PURPOSE.POTATO_FIELD:    (255,   0, 128),
-			BUILDING_PURPOSE.PASTURE:         (  0, 192,   0),
-			BUILDING_PURPOSE.WEAVER:          (  0,  64,  64),
+			BUILDING_PURPOSE.PASTURE:         (000, 192,   0),
+			BUILDING_PURPOSE.WEAVER:          (000,  64,  64),
 			BUILDING_PURPOSE.SUGARCANE_FIELD: (192, 192,   0),
 			BUILDING_PURPOSE.DISTILLERY:      (255, 128,  40),
-			BUILDING_PURPOSE.TOBACCO_FIELD:   ( 64,  64,   0),
+			BUILDING_PURPOSE.TOBACCO_FIELD:   (+64,  64,   0),
 			BUILDING_PURPOSE.TOBACCONIST:     (128,  64,  40),
-			BUILDING_PURPOSE.CLAY_PIT:        (  0,  64,   0),
-			BUILDING_PURPOSE.BRICKYARD:       (  0,  32,   0),
+			BUILDING_PURPOSE.CLAY_PIT:        (000,  64,   0),
+			BUILDING_PURPOSE.BRICKYARD:       (000,  32,   0),
 			BUILDING_PURPOSE.BOAT_BUILDER:    (163,  73, 164),
 			BUILDING_PURPOSE.SALT_PONDS:      (153, 217, 234),
-			BUILDING_PURPOSE.HERBARY:         ( 64, 200,   0),
-			BUILDING_PURPOSE.RESERVED:        (  0,   0, 128),
+			BUILDING_PURPOSE.HERBARY:         (+64, 200,   0),
+			BUILDING_PURPOSE.RESERVED:        (000,   0, 128),
```

Comment:
Please exclude the changes in this file from the PR, we'd like to keep in the old state for code layout reasons.